<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signupwindow</class>
 <widget class="QDialog" name="signupwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Management Register Page</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/pic/New folder/6681213.png</normaloff>:/img/pic/New folder/6681213.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="pushButton_signin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>301</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color:transparent;
border:none;
color: rgb(255, 10, 14)
}








</string>
   </property>
   <property name="text">
    <string>Already have an account ? Sign In</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>User Name :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(2, 29, 204);     /* Blue background */
    color: white;                  /* White text */
    border-radius: 10px;           /* Rounded corners */
    padding: 8px 16px;             /* Inner spacing */
    font-weight: bold;
   
}

QPushButton:hover {
    background-color: #0056b3;     /* Darker blue on hover */
}

QPushButton:pressed {
    background-color: #003d80;     /* Even darker on press */

}</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newpassword">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #007bff;   /* Blue border */
    border-radius: 5px;          /* Rounded corners */
    padding: 5px;                /* Inner spacing */
   
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter your password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_confirmpassword">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #007bff;   /* Blue border */
    border-radius: 5px;          /* Rounded corners */
    padding: 5px;                /* Inner spacing */
   
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Re-Enter password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Confirm Password :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newusername">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #007bff;   /* Blue border */
    border-radius: 5px;          /* Rounded corners */
    padding: 5px;                /* Inner spacing */
   
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter your username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Password :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>35</pointsize>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(239, 0, 4);</string>
   </property>
   <property name="text">
    <string>Create new account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>-50</y>
     <width>681</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/pic/New folder/1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>pushButton_signin</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_register</zorder>
  <zorder>lineEdit_newpassword</zorder>
  <zorder>lineEdit_confirmpassword</zorder>
  <zorder>lineEdit_newusername</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
